src/utils/options.ts

```diff
@@ -75,6 +77,14 @@ async function parseOptionsWithoutTryCatch(
         return { error: new Error(`expected value for option: ${arg}`) };
       }
       signifySecretKey = value;
```

Comment:
@benadida I left in the signify functionality here, but since I replaced it elsewhere should I take it out?

Reply:
done